release_notes.rst

```diff
@@ -14,9 +14,116 @@ NethServer 8 releases
+  Traefik "state/" directory in ``*.v2`` directories. These can be safely
+  removed if desired.
+
+  The Traefik restoration procedure has been fixed and no longer returns a
+  conflicting Traefik instance. Instead, the backup contents are merged
+  with the active Traefik instance on the node, allowing the restoration of
+  custom HTTP routes and TLS certificates.
+
+  Additionally, this release introduces two new advanced experimental
+  features, accessible through API calls and manual configuration:
+
+  1. Support for deploying NS8 behind a network HTTP L7 Proxy.
+  2. Passing extra parameters to the Traefik container to configure custom
+     CA certificates and DNS-01 challenges.
+
+  Refer to available support channels for more information.
+
+- **Per-IP access restrictions on HTTP routes** -- The ``HTTP routes``
+  page now allows access restrictions for automatic and manually-created
+  routes based on a list of IP addresses. Additionally, a ``cluster-admin``
+  HTTP route entry is now displayed and can be used to restrict access to
+  the Cluster Admin UI. Refer to the :ref:`traefik-section` for detailed
+  information.
+
+- **New Monitoring/Alarms stack** -- The Metrics core application,
+  including a running Prometheus instance, is now part of the default core
+  applications and is automatically installed on the leader node of existing
+  clusters with a simple core update. Metrics can be easily integrated
+  with other monitoring solutions and configured to send alert and
+  resolution notifications. Read further details in :ref:`metrics-section`.
+
+- **Migration tool enhancements** -- In addition to "Application
+  conflict management," previously released, the NS7 migration tool now
+  considers application instance conflicts, providing guidance to avoid
+  misconfigurations during migration.
+
+  Furthermore, for NethVoice migration, the NethVoice Proxy installation
+  is now handled automatically by the migration tool.
+
+- **Imapsync and Sieve filters** -- The Imapsync application can now be
+  configured to enable Sieve filter execution during "INBOX-only"
+  synchronization. See :ref:`imapsync-section` for more information.
+  Additionally, recursive synchronization has been optimized to reduce
+  system load when executing multiple concurrent Imapsync tasks.
+
+- **DNSMasq gateway option** -- The client gateway can now be set from the
+  :ref:`DHCP section <dnsmasq-dhcp-section>` of DNSMasq. Additionally,
+  selectable network interfaces are now limited to those with a private IP
+  address to prevent configuration errors.
+
+- **Webtop updates** -- Webtop has been updated to upstream release 5.27.3
+  with the new Pecbridge component version 5.4.5. The memory limit has been
+  raised to 4GB for better performance. This release also introduces
+  automated TinyMCE Plugin Integration with an active subscription and
+  enhanced contact sharing with the latest NethVoice application release.
+
```

Comment:
As soon as NethVoice is ready, we can add here a little paragraph linked to voice manual